strawberryfields/gbs/plot.py

```diff
@@ -345,49 +346,72 @@ def points(
     return f
 
 
-LIGHT_BLUE = "#94cecf"
-
-BAR_COLOUR = LIGHT_BLUE
-
-
-def spectrum(energies: list, bins: Optional[int] = 0):  # pragma: no cover
-    """Plots a spectrum based on input sampled energies.
+def spectrum(
+    energies: list, gamma: float = 100.0, xmin: float = None, xmax: float = None
+):  # pragma: no cover
+    """Plots a vibronic spectrum based on input sampled energies.
 
     Args:
         energies (list[float]): a list of sampled energies
-        bins (int): maximum number of bins. If zero, the number of bins is decided automatically
-            by Plotly.
+        gamma (float): parameter specifying the width of the Lorentzian function
```

Comment:
It might be worth explaining the Lorentzian function better in the docstring.

Reply:
We can do it in the polishing PRs.